charts/harvester-csi-driver/harvester-csi-driver/100.0.0+up0.1.5/templates/_helpers.tpl

```diff
@@ -0,0 +1,74 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
Even if we don't support windows we need toleraters/selectors for the linux nodes in our mixed node clusters.

Reply:
This is only thing left, then should be good